validate/boolean/src/index.ts

```
@@ -0,0 +1,141 @@
+    ? languageMappings[config.language]
+    : languageMappings.en
+  const mapping = config.customMapping || defaultMapping
+
+  let normalizedValue = value
+  if (typeof value === 'string' && !config.caseSensitive) {
+    normalizedValue = value.toLowerCase()
+  }
+
+  if (normalizedValue === true || normalizedValue === false) {
+    record.set(field, normalizedValue)
+  } else if (mapping.hasOwnProperty(normalizedValue)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)